src/main/java/net/starschema/clouddb/jdbc/BQResultSetMetadata.java

```diff
@@ -192,16 +187,24 @@ public String getColumnName(int column) throws SQLException {
 
     /**
      * {@inheritDoc} <br>
-     * note: This Can only Return due to bigquery:<br>
-     * java.sql.Types.DOUBLE<br>
-     * java.sql.Types.BOOLEAN<br>
-     * java.sql.Types.BIGINT<br>
-     * java.sql.Types.VARCHAR<br>
-     * java.sql.Types.TIMESTAMP<br>
-     * java.sql.Types.DATE<br>
-     * java.sql.Types.NUMERIC<br>
-     * java.sql.Types.RECORD<br>
-     * java.sql.Types.STRUCT<br>
+     * note: see below for BQ type => Java type enum<br>
+     * INTEGER => java.sql.Types.BIGINT<br>
+     * NUMERIC => java.sql.Types.NUMERIC<br>
+     * FLOAT => java.sql.Types.DOUBLE<br>
+     * BOOLEAN => java.sql.Types.BOOLEAN<br>
+     * STRING => java.sql.Types.VARCHAR<br>
+     * BYTES => java.sql.Types.VARCHAR<br>
+     * DATE => java.sql.Types.DATE<br>
+     * DATETIME => java.sql.Types.TIMESTAMP<br>
+     * TIME => java.sql.Types.TIME<br>
+     * TIMESTAMP => java.sql.Types.TIMESTAMP<br>
+     * ARRAY => unsupported<br>

```

Comment:
Note that arrays seem to be natively supported (selecting an ARRAY in the sql runner works fine)... and GEOGRAPHY just doesn't seem worth implementing.